hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/functional/TestHoodieBackedMetadata.java

```diff
@@ -1093,8 +1150,9 @@ public void testErrorCases() throws Exception {
       newCommitTime = HoodieActiveTimeline.createNewInstantTime();
       client.startCommitWithTime(newCommitTime);
       records = dataGen.generateInserts(newCommitTime, 5);
-      writeStatuses = client.bulkInsert(jsc.parallelize(records, 1), newCommitTime).collect();
+      writeStatuses = client.insert(jsc.parallelize(records, 1), newCommitTime).collect();
```

Comment:
Why not keep bulkInsert?

Reply:
nothing intentional. after an upsert, bulk insert did not make sense and hence switched.